src/app/tests/yaml/onoff.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
```

Comment:
nit on naming: yaml/foo.yaml stutters. Also describes format which could be ephemeral (it will likely not be as I see yaml more readable than xml or json or others, but who knows).

Suggest `src/app/tests/suites/onoff.yaml` or `src/app/tests/definitions/onoff.yaml`